packages/http/src/mocker/negotiator/types.ts

```diff
@@ -4,7 +4,7 @@ import { IHttpOperationConfig, JSONSchema } from '../../';
 
 export interface IHttpNegotiationResult {
   code: string;
-  mediaType: string;
+  mediaType?: string;
```

Comment:
Ah ok I understand now, if the request is `HEAD` there is no media type we're effectively returning.